src/Sentry/Protocol/IScope.cs

```diff
@@ -69,6 +58,17 @@ public interface IScope
         /// <remarks>Requires Sentry 8.0 or higher.</remarks>
         string? Environment { get; set; }
 
+        /// <summary>
+        /// The name of the transaction in which there was an event.
+        /// </summary>
+        /// <remarks>
+        /// A transaction should only be defined when it can be well defined.
+        /// On a Web framework, for example, a transaction is the route template
+        /// rather than the actual request path. That is so GET /user/10 and /user/20
+        /// (which have route template /user/{id}) are identified as the same transaction.
+        /// </remarks>
+        string? TransactionName { get; set; }
```

Comment:
Probably with a changelog item just for this change since it's a breaking one:
* Transaction on Scope and Event now is called TransactionName